src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -973,19 +958,6 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
-    // Method Description:
-    // - Returns true if we should exit the application before even starting the
-    //   window. We might want to do this if we're displaying an error message or
-    //   the version string, or if we want to open the settings file.
-    // Arguments:
-    // - <none>
-    // Return Value:
-    // - true iff we should exit the application before even starting the window
-    bool TerminalWindow::ShouldExitEarly()

```

Comment:
📝 check that we still handle this scenario

Reply:
This is handled by the following code in WindowEmperor.cpp:
```cpp
// If we created no windows, e.g. because the args are "/?" we can just exit now.
_postQuitMessageIfNeeded();
```

But thank you for bringing this up, because I just noticed that `wtd /?` does not work.